### PR TITLE
[ADD] elico_project_extra

### PR DESCRIPTION
**Pull request by noahzaozao**
@seb-elico @elicoidal Please review this PR , this is from superman but not installed in our ERP.

### CODE REVIEW

**Comment by seb-elico** [comment]
@noahzaozao @elicoidal this module is a mess, it has so many different purposes. Including some very Elico specific features (send an email to Eric for instance). I don't mind having it in our public addons (as long as we refactor it into several modules when we port it to v9) but we should remove the mentions to Elico related stuff

File: elico_project_extra/__init__.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
```
Comment (seb-elico) [inline elico_project_extra/__init__.py]: Short headers

**Comment by seb-elico** [comment]
Else we can keep it in `elico_odoo` but it's not even installed in our ERP :laughing:

**Review by seb-elico — CHANGES_REQUESTED** [review]

**Comment by elicoidal** [comment]
agree @seb-elico  this one is too messy to publish unless it is split. there is few interest in this effort.
We should have it in a repo private Elico

File: elico_project_extra/project.py [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+        task_id = self.browse(cr, uid, task_id, context=context)
+        
+        #vals['type_id']= task_id.project_id and task_id.project_id.type_ids and task_id.project_id.type_ids[-1].id
+        self.write(cr, uid, [task_id.id],vals, context=context)
+        
+        #LY send a email reminder to task responsible and Eric.
+        mail_message = self.pool.get('mail.message')
+        subject = '['+str(task_id.id)+'] "' + task_id.name + '" is finished'
+        rendered_body = "Task [%d] \"%s\" is Finished.\n\n Task Descriptions:\n %s \n\n Project: \"%s\".\n Progress: %%%.2f. \n\n-----\n Good Job. \n Project Task Reminder"%(task_id.id, task_id.name,  task_id.description, task_id.project_id.name, task_id.project_id.progress_rate or 100)
+        email_from = "eric.caudal@elico-corp.com"
```
Comment (seb-elico) [inline elico_project_extra/project.py]: Please change to `admin@example.com`

File: elico_project_extra/project.py [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+        email_to = [task_id.user_id.user_email]
+        
+        if task_id.user_id.user_email:
+            email_cc = ["eric.caudal@elico-corp.com"]
```
Comment (seb-elico) [inline elico_project_extra/project.py]: Please change to `admin@example.com`

File: elico_project_extra/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+# © 2015 Elico corp (www.elico-corp.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+
+{
+    'name': 'Project Extra module for Elico Corp',
```
Comment (seb-elico) [inline elico_project_extra/__openerp__.py]: Please change to `Project Extra module`

**Review by seb-elico — CHANGES_REQUESTED** [review]

**Comment by seb-elico** [comment]
@elicoidal @noahzaozao IMO let's put it in our public repo and remove mentions to Elico Corp in the code (I requested some changes for that matter)

**Comment by noahzaozao** [comment]
@seb-elico Done

**Comment by seb-elico** [comment]
@noahzaozao sorry I forgot to ask, can you rename the module to `project_extra`?

**Review by seb-elico — APPROVED** [review]

**Comment by noahzaozao** [comment]
@seb-elico Done

**Comment by seb-elico** [comment]
LGTM FT

**Pull request merged**